### PR TITLE
Refactor for proxy

### PR DESCRIPTION
**Pull request by LilCare**

### CODE REVIEW

File: client/cssModules/button.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+.shuffle-alt {
+    composes: button;
+    background-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCI+PHBhdGggZmlsbD0iI2Y1MCIgZD0iTTEzLjU4NiAxN2wtOC04SDNWN2gzLjQxNGw4IDhIMTd2MmgtMy40MTR6TTMgMTVoMi41ODZsMi4yMDctMi4yMDcgMS40MTQgMS40MTQtMi41MDEgMi41MDEtLjI5My4yOTJIM3YtMnptMTQtNmgtMi41ODZsLTIuMjA3IDIuMjA3LTEuNDE0LTEuNDE0TDEzLjU4NiA3SDE3djJ6bTQgN2wtNCAzdi02bDQgM3ptMC04bC00IDNWNWw0IDN6Ii8+PC9zdmc+Cg==);
+}
+
+.repeat {
+    composes: button;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24'%3E%3Cpath fill='%23333' d='M12 8H9a4 4 0 1 0 0 8h6a4 4 0 0 0 2.104-7.403l1.77-1.18.02.018A6 6 0 0 1 15 18H9A6 6 0 1 1 9 6h3V4l4 3-4 3V8z'/%3E%3C/svg%3E");
```
Comment (quetzalcaotl) [inline client/cssModules/button.css]: How come line 35 has the url in quotes while the other ones don't? Is this doing something special I don't see?

File: coverage/coverage-final.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (quetzalcaotl) [inline coverage/coverage-final.json]: I'm sure it doesn't matter, but is this document doing anything?

File: database/seed.js [hunk 3]
```diff
@@ -6,19 +6,19 @@ const seedData = () => {
...
     // If no song at that index, use the last song (uses faker data to fill in details)
     const randomSong = {
-      songId: Math.floor(Math.random() * 999),
-      length: Math.floor(Math.random() * (500 - 100) + 100),
+      songId: Math.floor(Math.random() * 2000),
+      length: 183,
       timestamp: 0,
       isliked: Math.floor(Math.random() * 2),
-      songfile: faker.internet.url(),
+      songFile: 'https://sound-clout.s3-us-west-1.amazonaws.com/Fleas.mp3',
```
Comment (quetzalcaotl) [inline database/seed.js]: Just to confirm: All of your database entries are playing the same song?

File: client/cssModules/button.css (hunk 1 above)
Comment (LilCare) [reply client/cssModules/button.css]: I used an encoder to create this url, so the format is different

File: coverage/coverage-final.json (hunk 2 above)
Comment (LilCare) [reply coverage/coverage-final.json]: It is related to testing.  It is in my .gitignore, but I think I need to delete for the files to not show up

File: database/seed.js (hunk 3 above)
Comment (LilCare) [reply database/seed.js]: Only the last few.  The first 12 play different ones, I just rounded it out to 15 by repeating one for the last few

Comment (LilCare) [reply database/seed.js]: (I was thinking I would create a nextUp pop up, and wanted to have a decent number to scroll through)

**Pull request merged**